.travis.yml

```diff
@@ -1,11 +1,20 @@
-os:
-- linux
-- osx
+
+matrix:
+  include:
+    - os: linux
+      sudo: required
+      compiler: gcc
+    - os: linux
+      sudo: required
+      compiler: clang
+    - os: osx
+      compiler: gcc
+    - os: osx
+      compiler: clang
 
 language: c
 
-compiler:
-- clang
-- gcc
+before_install:
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo pip install scikit-ci-addons==0.15.0; sudo ci_addons travis/install_cmake 3.2.0; fi
```

Comment:
Why are we *sudo* pip installing scikit-ci-addons? This is going to make our builds _incredibly_ slow (the use of sudo)